frontend/packages/kubevirt-plugin/src/components/vm-templates/instantiate-template/TemplateForm.tsx

```diff
@@ -45,6 +50,8 @@ export const TemplateForm: React.FC<TemplateFormProps> = ({
   const [inProgress, setInProgress] = React.useState<boolean>(false);
   const [errorMsg, setErrorMsg] = React.useState<string>('');
 
+  const isCreateTemplate = mode === VMWizardMode.TEMPLATE;
```

Comment:
consider using useMemo, it will recalc every render

Reply:
added thanks :)